src/style.css

```diff
@@ -1 +1,78 @@
 /* Styles go here */
+
+* {
```

Comment:
usually this selector is used only to add a `box-sizing: border-box;` rule
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/a2dc5461-f8ea-4585-a3ed-1e7ea8c26c94)